Tools/Scripts/build-webkit

```diff
@@ -364,6 +364,10 @@ if (isAppleWinWebKit() || isWinCairo() || isPlayStation() || isFTW()) {
     push @local_options, XcodeCoverageSupportOptions() if $coverageSupport;
     push @local_options, XcodeStaticAnalyzerOption() if $shouldRunStaticAnalyzer;
     push @local_options, "WK_LTO_MODE=$ltoMode" if ($ltoMode ne "default");
+    if (isAppleMacWebKit()) {
+        my $version = osXVersion();
+        push @local_options, "MACOSX_DEPLOYMENT_TARGET=$version->{major}.$version->{minor}";
+    }
```

Comment:
What about people who don't use build-webkit?

Can't we do this in the xcconfig files instead?

Reply:
Not easily, because (afaik) there isn't a good way to get the `x.y` version number. There's $(MAC_OS_X_VERSION_MINOR), but it's integer formatted (i.e. `MAC_OS_X_VERSION_MINOR = 130300`), and I don't think xcconfig has a way to break that down into a decimal-separated version number.